src/WebJobs.Script/Binding/Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -79,6 +80,11 @@ public HttpTriggerBinding(HttpTriggerAttribute attribute, ParameterInfo paramete
                     }
                 }
 
+                if(attribute.IsProxy)
```

Comment:
Not sure if I get this. IsProxy is property of attribute. _isProxy is a private local field.